data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/main/ConsumerVerticleFactoryImpl.java

```diff
@@ -168,7 +169,7 @@ public AbstractVerticle get(final DataPlaneContract.Resource resource, final Dat
           filter,
           egressSubscriberSender,
           egressDeadLetterSender,
-          new HttpSinkResponseHandler(vertx, resource.getTopics(0), producer),
+          new KafkaResponseHandler(producer, resource.getTopics(0)),
```

Comment:
why don't we provide the `vertx` instance to the ResponseHandler anymore?

Reply:
Simply, we don't need it anymore (this was probably required before for the tracing old impl)